iac/main.tf

```diff
@@ -26,4 +26,9 @@ locals {
 resource "azurerm_resource_group" "resource_group" {
   name     = local.resource_group_name
   location = var.location
+
+  tags = {
+    projectName = "Demo"
+    owner = "Facundo"
```

Comment:
Could this be my full name `Facundo Gauna`?

Reply:
Sorry, I took the WI too literally.

Reply:
No worries!